lib/rack/auth/digest/md5.rb

```diff
@@ -98,7 +99,7 @@ def valid_nonce?(auth)
 
         def valid_digest?(auth)
           pw = @authenticator.call(auth.username)
-          pw && Rack::Utils.secure_compare(digest(auth, pw), auth.response)
+          pw && Utils.secure_compare(digest(auth, pw), auth.response)
```

Comment:
This is an example of the mentioned **Secondary Change**. Even in this one file we reference the `Utils` constant inconsistently. I can be convinced that either fully qualified or module-relative is the way to go for Rack. But we should be consistent, whichever way we go.

Reply:
I like this change.